cmd.go

```diff
@@ -60,6 +68,19 @@ func infocmd(opt Options) error {
 		releases = []Release{*release}
 	}
 
+	if err := renderer(tags, releases); err != nil {
+		return err
+	}
+
+	return nil
```

Comment:
Agreed - it *can* be reduced to that.  For me this comes down to preferring to optimize for making it easy to debug return values without having to create more variables.

With that said, this is your repo and project.  I'm happy to concede to your preferences here!